src/utils/searchUtils.ts

```diff
@@ -1,6 +1,9 @@
 import { isArray, isEmpty, mapObject, forEach } from "underscore"
 
-import { textInputFields as advSearchFields } from "./advancedSearchUtils"
+import {
+  textInputFields as advSearchFields,
```

Comment:
Does textInputFields need to be imported twice?

Reply:
removed!